elide-core/src/main/java/com/yahoo/elide/core/PersistentResource.java

```diff
@@ -786,12 +793,23 @@ public PersistentResource getRelation(String relation, String id) {
         Set<PersistentResource<Object>> resources = Sets.newLinkedHashSet();
         if (val instanceof Collection) {
             Collection filteredVal = (Collection) val;
+            // sorting/pagination supported on last entity only eg /v1/author/1/books? books would be valid
+            final boolean hasSortRules = !requestScope.getSorting().isDefaultInstance();
+            final boolean isPaginated = !requestScope.getPagination().isDefault();
 
-            if (!filters.isEmpty()) {
+            if (!filters.isEmpty() || hasSortRules || isPaginated) {
                 final Class<?> entityClass = dictionary.getParameterizedType(obj, relationName);
-                filteredVal = requestScope.getTransaction().filterCollection(filteredVal, entityClass, filters);
+                final Optional<Sorting> sortingRules = hasSortRules ? Optional.of(requestScope.getSorting())
+                        : Optional.empty();
+                final Optional<Pagination> pagination = isPaginated ? Optional.of(requestScope.getPagination())
+                        : Optional.empty();
+                if (!filters.isEmpty()) {
```

Comment:
I don't understand this.  Why wouldn't we always call filterSortOrPaginate here.